python/python/Ice/IcePyTypes.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) ZeroC, Inc.
+
+# These are the public types we re-export from IcePy.
+
+from IcePy import (
+    BatchRequest,
+    Connection,
+    ConnectionInfo,
+    EndpointInfo,
+    IPConnectionInfo,
+    IPEndpointInfo,
+    NativePropertiesAdmin,
+    OpaqueEndpointInfo,
+    SSLConnectionInfo,
+    SSLEndpointInfo,
+    TCPConnectionInfo,
+    TCPEndpointInfo,
+    UDPConnectionInfo,
+    UDPEndpointInfo,
+    WSConnectionInfo,
+    WSEndpointInfo,
+    currentEncoding,
+    currentProtocol,
+    currentProtocolEncoding,
+    encodingVersionToString,
+    intVersion,
+    loadSlice,
+    protocolVersionToString,
+    stringToEncodingVersion,
+    stringToProtocolVersion,
+    stringVersion,
+)
+
+__all__ = [
+    "BatchRequest",
+    "Connection",
+    "ConnectionInfo",
+    "EndpointInfo",
+    "IPConnectionInfo",
+    "IPEndpointInfo",
+    "NativePropertiesAdmin",
+    "OpaqueEndpointInfo",
+    "SSLConnectionInfo",
+    "SSLEndpointInfo",
+    "TCPConnectionInfo",
+    "TCPEndpointInfo",
+    "UDPConnectionInfo",
+    "UDPEndpointInfo",
+    "WSConnectionInfo",
+    "WSEndpointInfo",
+    "currentEncoding",
```

Comment:
Starting with currentEncoding: I don't see the point of reexporting or documenting any of these symbols in Python, with the exception of `loadSlice, `intVersion` and `stringVersion`.

In particular, all the Protocol symbols are totally useless: the protocol is always "1.0".